examples/lib/lib_test.go

```diff
@@ -24,3 +24,7 @@ func TestMeaning(t *testing.T) {
 		t.Errorf("got %d, want 42", m)
 	}
 }
+
+// func TestAddTwoNumbers(t *testing.T) {
```

Comment:
github.com/bazelbuild/rules_go/examples/lib/lib_test.AddTwoNumbers: call to external function github.com/bazelbuild/rules_go/examples/lib/lib_test.add
github.com/bazelbuild/rules_go/examples/lib/lib_test.AddTwoNumbers: github.com/bazelbuild/rules_go/examples/lib/lib_test.add: not defined
github.com/bazelbuild/rules_go/examples/lib/lib_test.AddTwoNumbers: undefined: github.com/bazelbuild/rules_go/examples/lib/lib_test.add